src/vl-wizard-pane.js

```diff
@@ -70,7 +74,8 @@ export class VlWizardPane extends VlElement(HTMLElement) {
      * @return {String}
      */
     get title() {
-        return this._titleSlot ? this._titleSlot.innerText : undefined;
+        const element = this._progressBarTitleSlot || this._titleSlot;
```

Comment:
Beetje raar om maar 1 getter te hebben terwijl er 2 titels zijn.

Reply:
Deze functie geeft de titel terug van het panel en zal eerst kijken naar het progress bar title slot element en als fallback het title slot element gebruiken. Een alternatief zou kunnen zijn om in de wizard een fallback te zetten en de 2 afzonderlijke title getters aan te roepen, maar het lijkt me properder dat de wizard de titel opvraagt bij het panel die vervolgens zelf bepaalt wat de fallback is.

Tenzij jullie nood hebben aan een getter voor de afzonderlijke titels, lijkt het me niet nodig dit aan te passen.

Reply:
Hebben we niet nodig, had ik gewoon opgemerkt in onze testen waar we deze getter gebruiken...

Reply:
Op welk niveau zijn jullie dit aan het (her)testen, unit of e2e?

Reply:
In een frontend unit test waar wij valideren dat we de (dynamische) titel correct zetten.

Reply:
De webcomponenten worden grondig getest op unit en e2e niveau, hertesten op applicatie niveau zou zonde zijn. Uiteraard kunnen configuratie testen wel relevant zijn.